### PR TITLE
paginated list layout: made number of pages shown in paginated bar

### PR DESCRIPTION
**Pull request by mhauck**
configurable

### CODE REVIEW

File: solute/epfl/components/paginated_list_layout/paginated_list_layout.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ class PaginatedListLayout(PrettyListLayout):
     show_pagination = True  #: Set to true to show the pagination bar.
     show_search = True  #: Set to true to enable the search field.
     search_placeholder = "Search..."  #: Placeholder text for the search input.
+    number_of_pagination_pages = 5  #: Specify the number of pages that should be visible in the pagination bar.
```
Comment (JustusW) [inline solute/epfl/components/paginated_list_layout/paginated_list_layout.py]: visible_pages_limit is probably a better choice.

File: solute/epfl/components/paginated_list_layout/theme/after.html [hunk 2]
```diff
@@ -4,9 +4,9 @@
     {% if compo.row_count > compo.row_limit and compo.show_pagination %}
     <div class="col-sm-12 text-center">
         <ul class="pagination-sm pagination" id="{{ compo.cid }}_pagination" style="margin: 5px 0; ">
-        	{% set current_page = (compo.row_offset/compo.row_limit)|round(method='floor')|int %}
+            {% set current_page = (compo.row_offset/compo.row_limit)|round(method='floor')|int %}
             {% set max_page = (compo.row_count/compo.row_limit)|round(method='floor')|int %}
-            {% if max_page > 4 %}
+            {% if max_page > (compo.number_of_pagination_pages-1) %}
```
Comment (JustusW) [inline solute/epfl/components/paginated_list_layout/theme/after.html]: spaces before and after - please.

File: solute/epfl/components/paginated_list_layout/theme/after.html [hunk 3]
```diff
@@ -14,30 +14,31 @@
...
-	        {% endfor %}
+                <li class="{{ 'active' if i == compo.row_offset else '' }}">
+                    <span>{{ (i/compo.row_limit + 1)|round(method='ceil')|int }}</span>
+                </li>
+            {% endfor %}
             <li class="{{ '' if compo.row_offset + compo.row_limit < compo.row_count else 'disabled' }}">
                 <span aria-hidden="true">&gt;</span>
             </li>
-            {% if max_page > 4 %}
+            {% if max_page > (compo.number_of_pagination_pages-1) %}
```
Comment (JustusW) [inline solute/epfl/components/paginated_list_layout/theme/after.html]: spaces before and after - please.

**Comment by mhauck** [comment]
Comments are fixed.

**Pull request merged**